src/waitForAxeResults.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { expect, test } from "@playwright/test"
+import { readFile } from "./utils/file"
+import { waitForAxeResults } from './waitForAxeResults'
+
+test.describe("waitForAxeResults", () => {
+  test("should be ok for page with no axe violations", async ({ page }) => {
+    const content = await readFile("accessible.html")
+    await page.setContent(content)
+    const { ok } = await waitForAxeResults(page)
+    await expect(ok).toBeTruthy()
```

Comment:
Always prefer `toBe(true)` as it tests that an actual boolean was returned.

```suggestion
    const { ok } = await waitForAxeResults(page)
    await expect(ok).toBe(true)
```

Reply:
Additionally, could we test the results object that is returned?